bot.go

```diff
@@ -36,7 +37,9 @@ func NewBot(f ...func(*Bot)) *Bot {
 		Data:      make(chan *irc.Message),
 		callbacks: make(map[string][]Callback),
 		tries:     0,
+		stop:      make(chan bool),
 	}
+	b.stop <- false
```

Comment:
We probably don't want to send here, but we should listen in the `Disconnect` method. 

We could also avoid the use of an external channel completely and just call the `Disconnect` method, and listen on it internally.